frontend/internal-packages/db/supabase/migrations/20250616065500_add_validation_tables.sql

```diff
@@ -0,0 +1,281 @@
+BEGIN;
+
+-- Create validation_queries table
+CREATE TABLE IF NOT EXISTS "public"."validation_queries" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "design_session_id" "uuid" NOT NULL,
+    "query_string" "text" NOT NULL,
+    "organization_id" "uuid" NOT NULL,
+    "created_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL,
+    "updated_at" timestamp(3) with time zone DEFAULT CURRENT_TIMESTAMP NOT NULL
+);
+
+ALTER TABLE "public"."validation_queries" OWNER TO "postgres";
+
+-- Create validation_results table
+CREATE TABLE IF NOT EXISTS "public"."validation_results" (
+    "id" "uuid" DEFAULT "gen_random_uuid"() NOT NULL,
+    "validation_query_id" "uuid" NOT NULL,
+    "result_set" "text",
```

Comment:
(aside)
(ask)

What kind of data does result_set contain?
I'm asking because if it's similar to a SQL query result, I was thinking it might be representable as a JSON or JSONB array.